lib/commons/aria/index.js

```diff
@@ -2385,45 +2385,3 @@ lookupTable.evaluateRoleForElement = {
 		return out;
 	}
 };
-
-/**
- * Reference -> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Widget_roles
- * The current lookupTable.role['widget'] widget, yeilds
- * ->
- * [
- * 	"alert", "alertdialog", "button", "checkbox", "dialog", "gridcell", "link", "log", "marquee", "menuitem", "menuitemcheckbox",
- * 	"menuitemradio", "option", "progressbar", "radio", "scrollbar", "searchbox", "slider", "spinbutton", "status", "switch", "tab", "tabpanel",
- * 	"textbox", "timer", "tooltip", "treeitem"
- * ]
- * There are some differences against specs, hence the below listing was made
- */
-lookupTable.rolesOfType = {
```

Comment:
Taking this out could break custom rules. Please leave it in, label it as deprecated, and make sure to include the word "deprecated" in the title of the PR so it appears in the changelog.